.github/workflows/instrumentation_tests.yml

```diff
@@ -33,19 +33,20 @@ jobs:
         if: failure()
         with:
           name: instrumentation-test-report
-          path: example/build/reports/androidTests/connected/index.html
+          path: example/build/reports/androidTests/connected/index.html | paymentsheet-example/build/reports/androidTests/connected/index.html
```

Comment:
This doesn't fetch any of the screenshot you generate right? Maybe we could figure out how to pull those too.